tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -82,11 +82,15 @@ sub run_test {
 
             #detach the vf from host
             $vf{host_bdf} = $host_vfs[int(rand($#host_vfs + 1))];
-            for (my $j = 0; $j < $i; $j++) {
+            my $j = 0;
+            while ($j < $i) {
```

Comment:
Maybe more meaningful name of j, i can be more readable.

Reply:
Add a piece of comment to make it clearer.